mariadb.spec

```diff
@@ -891,6 +900,10 @@ rm -rf %{buildroot}%{_datadir}/%{pkg_name}/solaris/
 # rename the wsrep README so it corresponds with the other README names
 mv Docs/README-wsrep Docs/README.wsrep
 
+# rename the wsrep_sst_rsync_tunnel README and license
+mv %{wsrep_sst_rsync_tunnel_path}/README.md %{wsrep_sst_rsync_tunnel_path}/README.wsrep_sst_rsync_tunnel
```

Comment:
Not a blocker, I would just appreciate if readme would include definition of SST when first mentioning it.
Of course jfgi worked: http://galeracluster.com/documentation-webpages/glossary.html#term-state-snapshot-transfer